### PR TITLE
mem-hotplug: update user-guide

### PR DESCRIPTION
**Pull request by acardace**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
Update memory hotplug user guide
```

### CODE REVIEW

File: docs/compute/memory_hotplug.md [hunk 1]
```diff
@@ -3,8 +3,12 @@
 Memory hotplug was introduced in KubeVirt version 1.1, enabling the dynamic resizing of the amount of memory available to a running VM.
 
 ## Limitations
-* Memory hotplug is currently only supported on the x86_64 architecture.
+* Memory hotplug is currently only supported on the x86_64,arm64 architectures.
+* Linux guests running at least Linux v5.8 are fully supported.
+* Windows guests support has been added to virtio-win, but it should be considered unstable.
 * Current hotplug implementation involves live-migration of the VM workload.
+* VirtualMachines must have at least 1GiB of memory to support memory-hotplug.
```
Comment (vladikr) [inline docs/compute/memory_hotplug.md:10]: Is this limitation listed anywhere else (QEMU/Libvirt documentation)? I failed to find any reference.

**Review by vladikr — COMMENTED** [review]
One question in-line.

**Comment by vladikr** [comment]
/approve
Looks good to me in general, I only have one question about the 1Gi guest mem size requirement.

File: docs/compute/memory_hotplug.md (hunk 1 above)
Comment (vladikr) [reply docs/compute/memory_hotplug.md:10]: @acardace I see... I didn't review this therefore I wasn't aware of this change. I do wonder about the validating webhook. I'd expect us to simply require a restart via a condition.

**Review by jean-edouard — APPROVED** [review]

**Pull request merged**